pkg/scheduler/eventhandlers.go

```diff
@@ -571,6 +574,10 @@ func nodeSpecUnschedulableChanged(newNode *v1.Node, oldNode *v1.Node) bool {
 	return newNode.Spec.Unschedulable != oldNode.Spec.Unschedulable && !newNode.Spec.Unschedulable
 }
 
+func nodeAnnotationsChanged(newNode *v1.Node, oldNode *v1.Node) bool {
+	return !reflect.DeepEqual(oldNode.GetAnnotations(), newNode.GetAnnotations())
```

Comment:
use `equality.Semantic.DeepEqual` instead.

Reply:
@amewayne @alculquicondor 

I guess we have to do this in others as well, right? (nodeTaintsChanged etc)

Reply:
yes